docs/getting-started.md

```diff
@@ -1,58 +1,76 @@
-# Software Setup
-
-## Table Of Contents
+# Table of Contents
 <!-- 
     NOTE: when creating or re-creating a table of contents like this, you can
     save a LOT of time by using this tool: 
     https://github.com/ekalinin/github-markdown-toc
 -->
-- [Software Setup](#software-setup)
-  - [Table Of Contents](#table-of-contents)
-  - [Introduction](#introduction)
-  - [Installation and Setup](#installation-and-setup)
-  - [Operating Systems](#operating-systems)
-  - [Getting the Code](#getting-the-code)
-  - [Running the setup scripts](#running-the-setup-scripts)
-    - [Installing Software Dependencies](#installing-software-dependencies)
-    - [Installing an IDE](#installing-an-ide)
-      - [CLion](#clion)
-        - [Getting your Student License](#getting-your-student-license)
-        - [Installing CLion](#installing-clion)
-      - [VSCode](#vscode)
-  - [Building and Running the Code](#building-and-running-the-code)
-    - [Building from the command-line](#building-from-the-command-line)
-         - [Using the fuzzy finder](#using-the-fuzzy-finder)
-         - [Editing with Vim or NeoVim](#editing-with-vim-or-neovim)
-    - [Building with CLion](#building-with-clion)
-    - [With VSCode](#with-vscode)
-    - [Running our AI, Simulator, SimulatedTests or Robot Diagnostics](#running-our-ai-simulator-simulatedtests-or-robot-diagnostics)
-    - [Running AI vs AI](#running-ai-vs-ai)
-  - [Debugging](#debugging)
-    - [Debugging with CLion](#debugging-with-clion)
-    - [Debugging from the Command line](#debugging-from-the-command-line)
-  - [Profiling](#profiling)
-  - [Building for Jetson Nano](#building-for-jetson-nano)
-  - [Deploying Robot Software to the Jetson Nano](#deploying-to-jetson-nano)
-  - [Setting up Virtual Robocup 2021](#setting-up-virtual-robocup-2021)
-    - [Setting up the SSL Simulation Environment](#setting-up-the-ssl-simulation-environment)
-    - [Pushing a Dockerfile to dockerhub](#pushing-a-dockerfile-to-dockerhub)
+* [Software Setup](#software-setup)
+   * [Introduction](#introduction)
+   * [Installation and Setup](#installation-and-setup)
+      * [Operating Systems](#operating-systems)
+      * [Getting the Code](#getting-the-code)
+      * [Running the setup scripts](#running-the-setup-scripts)
+         * [Installing Software Dependencies](#installing-software-dependencies)
+         * [Installing an IDE](#installing-an-ide)
+            * [CLion](#clion)
+               * [Getting your Student License](#getting-your-student-license)
+               * [Installing CLion](#installing-clion)
+            * [VSCode](#vscode)
```

Comment:
I made the table of contents more generic. I experimented with layers of dropdowns but that became too ugly so I kept the dropdown to only be one layer so that we don't need to have a detailed table of contents